pages/index.js

```diff
@@ -1,30 +1,73 @@
-import React from "react";
+import React, { useRef, useState} from 'react';
 
 import TodoInput from "../components/todo-input";
 import TodoList from "../components/todo-list";
 
 import styled from "styled-components";
```

Comment:
```suggestion
import React, { useState} from 'react';
import styled from "styled-components";

import TodoInput from "../components/todo-input";
import TodoList from "../components/todo-list";
```
1. import문 작성할 때 외부 라이브러리들과 직접 작성한 컴포넌트들을 구분해주시면 가독성이 좋아집니다!
2. useRef는 왜 쓰신 걸까요?